vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -14,11 +14,11 @@ The Hub allows users to:
 - Perform business-level assessment of component readiness. 
 
 An Entando Application can make use of an Entando Hub in several ways:
-* The Local Hub, included in the Entando App Builder, displays a collection of components ready for use. These can be used to compose an application or as a starting point to create new components.	
-* Entando Cloud Hub is the public catalog containing packaged business capabilities and components provided by Entando and its partners throughout the world.
-* An enterprise Entando Hub, developed and curated by Entando clients and partners, can be used to share components within their organization or made available  for public use.
+* The **Local Hub**, included in the Entando App Builder, displays a collection of components ready for use to compose an application or as a starting point to create new implementations.	
+* **Entando Cloud Hub** is the public catalog containing packaged business capabilities and components provided by Entando and its partners throughout the world.
+* An **enterprise Entando Hub**, developed and curated by Entando clients and partners, can be used to share components within their organization or made available for public use.
```

Comment:
"their" doesn't work here; clients and partners aren't part of a single organization -> "their respective organizations"